apps/projects/contracts/Projects.sol

```diff
@@ -23,95 +75,224 @@ contract Projects is AragonApp {
         bytes32 repo;  // This is the internal repo identifier
         uint256 number; // May be redundant tracking this
         bool hasBounty;
-        uint256 bountySize;
-        address bountyWallet; // Not sure if we'll have a way to "retrieve" this value from status open bounties
+        uint standardBountyId;
     }
 
     // The entries in the registry.
     mapping(bytes32 => GithubRepo) repos;
 
     // Gives us an array so we can actually iterate
-    bytes32[] repoIDs;
+    bytes32[] repoIds;
 
     // Fired when a repository is added to the registry.
-    event RepoAdded(bytes32 id);
+    event RepoAdded(bytes32 repoId);
     // Fired when a repository is removed from the registry.
-    event RepoRemoved(bytes32 id);
+    event RepoRemoved(bytes32 repoId);
     // Fired when a bounty is added to a repo
-    event BountyAdded(bytes32 owner, bytes32 repo, uint256 issueNumber, uint256 bountySize);
+    event BountyAdded(bytes32 owner, bytes32 repoId, uint256 issueNumber, uint256 bountySize);
+    // Fired when fulfillment is accepted
+    event FulfillmentAccepted(bytes32 repoId, uint256 issueNumber, uint fulfillmentId);
```

Comment:
If the "primary key' is all three of these values would it make sense to create a unique PK that maps into a struct of these three values?